### DISCUSSION TITLE
Artifact Container failed... How do i fixed it?

### DISCUSSION THREAD

**Discussion by SathishGandhi9** [community/community#61046] [Actions]
### Select Topic Area

General

### Body

I have below error on build my project in workflow action.... How can i fixed it?  
Create Artifact Container failed: **Artifact storage quota has been hit. Unable to upload any new artifacts**

**Comment by h7x4** [comment]
This error means that you have hit your storage limit.

You can do either of the following:

- [Delete some workflow artifacts](https://docs.github.com/en/github-ae@latest/actions/managing-workflow-runs/removing-workflow-artifacts). (Do also note that if you have not configured otherwise, [GitHub will delete your artifacts after 90 days](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#artifact-and-log-retention-policy))
- [Upgrade your github plan or activate billing](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#included-storage-and-minutes). This will let you create more artifacts.
- [Open source your project](https://github.com/orgs/community/discussions/26438). GitHub does not restrict storage for open projects.
  
See also https://github.com/orgs/community/discussions/26542, and https://github.com/c-hive/gha-remove-artifacts by extension. It will make it easier to mass delete artifacts.

**Reply by vhs0s0** [reply]
Thanks, Changing the repository's visibility worked
